web/store/module.store.ts

```diff
@@ -429,6 +430,7 @@ export class ModulesStore implements IModuleStore {
       runInAction(() => {
         set(this.moduleMap, [moduleId, "link_module"], linkModules);
       });
+      this.fetchModuleDetails(workspaceSlug, projectId, moduleId);
```

Comment:
We don't have to refetch as this is just delete. We can remove the deleted record from local state.

Remove it from moduleMap after successful delete.